scripts/src/primazactl/cmd/worker/create/namespace.py

```diff
@@ -67,11 +67,30 @@ def add_args_namespace(parser: argparse.ArgumentParser):
         required=False,
         help="Config file containing agent roles")
 
+    parser.add_argument(
+        "-n", "--namespace",
+        dest="namespace",
+        type=kubernetes_name,
+        required=False,
+        help=f"namespace to create. Default: \
+            primaza-{type}",
+        default=f"primaza-{type}")
+
+    parser.add_argument(
+        "-s", "--main-namespace",
+        dest="main_namespace",
+        type=kubernetes_name,
+        required=False,
+        help=f"namespace of primaza main. Default: \
+            {PRIMAZA_NAMESPACE}",
+        default=PRIMAZA_NAMESPACE)
```

Comment:
I'd propose to rename this field `tenant`

```suggestion
    parser.add_argument(
        "-t", "--tenant",
        dest="tenant",
        type=kubernetes_name,
        required=False,
        help=f"primaza's tenant. Default: \
            {PRIMAZA_NAMESPACE}",
        default=PRIMAZA_NAMESPACE)
```

Reply:
Agreed with @filariow to handle this in a separate PR.